src/components/pool_flux_chart.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <section>
+    <div id="chart-container" />
+  </section>
+</template>
+<script>
+import * as d3Base from 'd3';
+export default {
+  name: "PoolFluxChart",
+    components: {
+    },
+    data() {
+    return {
+      publicPath: process.env.BASE_URL, // this is need for the data files in the public folder, this allows the application to find the files when on different deployment roots
+      d3: null,
+
+      // dimensions
+      w: null,
+      h: null,
+      margin: { top: 10, right: 25, bottom: 20, left: 200 },
+      chartWidth: null,
+      chartHeight: null,
+      svgChart: null,
+      chartContainer: null,
+      tooltip: null,
+      }
+  },
+  mounted(){      
+      this.d3 = Object.assign(d3Base);
+
+    // chart elements
+    this.w = document.getElementById("chart-container").offsetWidth;
+    this.h = document.getElementById("chart-container").offsetHeight;
+    this.chartWidth = this.w - this.margin.left - this.margin.right;
+    this.chartHeight = this.h - this.margin.top - this.margin.bottom;
+    this.chartContainer = this.d3.select("#chart-container")
+    
+    // define div for tooltip
+    // this.tooltip = this.chartContainer
+    //   .append("div")
+    //   .attr("class", "tooltip")
+
+    // create svg that will hold chart
+    this.svgChart = this.chartContainer
+      .append("svg")
+        .classed("chart", true)
+        .attr("viewBox", "0 0 " + (this.chartWidth + this.margin.left + this.margin.right) + " " + (this.chartHeight + this.margin.top + this.margin.bottom))
```

Comment:
I think setting width and heihgt attributes here using relative values should allow scaling with the page, may need to add css as well

Reply:
Cool, I'll take a look in the next round of edits